app/presenters/leaderboard_presenter.rb

```diff
@@ -0,0 +1,15 @@
+class LeaderboardPresenter
+  include ActionView::Helpers::NumberHelper
+
+  def top_five
+    Vote.most_scenic_places.map { |result| PlaceWithRating.new(result) }
+  end
+
+  def bottom_five
+    Vote.least_scenic_places.map { |result| PlaceWithRating.new(result) }
+  end
+
+  def percentage_coverage
+    number_with_precision((Place.count.to_f / 229334 * 100), precision: 2, strip_insignificant_zeros: true)
```

Comment:
What is `229334` here? Seems like a magic number. Can we have it expressed as a constant if it's needed for any reason?

Reply:
Good spot! It's from here: https://github.com/thedatascilab/ScenicOrNot/blob/master/web/app/top10.php#L58 and I was meaning to make a card to look at the stats and how they're calculated. The old page was computing a few other stats, but they're not currently displayed. But also to reexamine how this percentage coverage was arrived at, and if we need to change anything.

Reply:
And it turns out it has somehow changed - the Wikipedia page now lists `241,930` as the total land surface. (Did the UK expand since 2009? 😳 )